django/test/utils.py

```diff
@@ -152,7 +154,8 @@ def teardown_test_environment():
     del mail.outbox
 
 
-def setup_databases(verbosity, interactive, keepdb=False, debug_sql=False, parallel=0, aliases=None, **kwargs):
+def setup_databases(verbosity, interactive, time_keeper, keepdb=False, debug_sql=False, parallel=0,
+                    aliases=None, **kwargs):
```

Comment:
Adding `time_keeper` as the 3rd argument can cause issues for people that rely on the current signature and do not use keyword args.